apps/gportal/vite.config.js

```diff
@@ -1,9 +1,11 @@
 import { defineConfig } from "vite";
 import vue from "@vitejs/plugin-vue";
 
-export default defineConfig(() => {
-  require("dotenv").config({ path: `./.env` });
+const host="https://emx2.dev.molgenis.org";
+const schema="fdh";
```

Comment:
Might be good to use one of the default schema's or make this one of them. Since the server gets wiped every once in a while, using a non-default might lead to schema not found issues.

Reply:
This is not a major issue as we want to force the app to have the latest version of the data model (if there are any changes). For most dev work, we use the GDI instance rather than the emx2 dev so the model is stable. For this PR, we needed to work with the dev instance since we didn't want to update the GDI server.

Reply:
I will also create a new PR with my proxy config.